pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -83,6 +83,14 @@ tasks:
       connections.outputName: diffimPlots
       connections.fakesType: parameters.fakesType
       connections.coaddName: parameters.coaddName
+      atools.dipoleQuiver: DiffimSpatialMetricsDipoleQuiverPlot
+      atools.dipoleQuiver.angleName: dipole_direction
+      atools.dipoleQuiver.lengthName: dipole_separation
+      atools.dipoleQuiver.produce.plot.qKeyLabel: "0.2 arcsec Avg Dipole length"
```

Comment:
Is this right my read of `ax.quiverkey(q, 0.5, 0.9, 0.1, self.qKeyLabel, labelpos="E", coordinates="figure")` is that the quiver label is 0.1 arcsec.

Reply:
Now we include another parameter to set up this size accordingly.